packages/react/src/time-input/TimeInput.test.tsx

```diff
@@ -0,0 +1,132 @@
+import '@testing-library/jest-dom';
+import userEvent from '@testing-library/user-event';
+import 'html-validate/jest';
+import { useState } from 'react';
+import { axe, toHaveNoViolations } from 'jest-axe';
+import { cleanup, render, screen } from '../../../../test-utils';
+import { TimeInput, type TimeInputProps, type TimeValue } from './TimeInput';
+
+expect.extend(toHaveNoViolations);
+
+afterEach(cleanup);
+
+type ControlledTimeInputProps = Omit<TimeInputProps, 'label'>;
+
+function renderTimeInput(props?: ControlledTimeInputProps) {
+	return render(<ControlledTimeInput {...props} />);
+}
+
+function ControlledTimeInput({
+	onChange: onChangeProp,
+	value: valueProp,
+	...props
+}: ControlledTimeInputProps) {
+	const [value, setValue] = useState<TimeValue | undefined>(valueProp);
+
+	function onChange(value: TimeValue) {
+		setValue(value);
+		onChangeProp?.(value);
+	}
+
+	return (
+		<TimeInput label="Time" onChange={onChange} value={value} {...props} />
+	);
+}
+
+async function getInput() {
+	const el = await screen.getByRole('textbox');
+	expect(el).toBeInstanceOf(HTMLInputElement);
```

Comment:
I'm curious about the need for this assertion? It feels like it's stepping into implementation testing territory but I might be missing something.

Reply:
All of these are copy pastes from existing tests 🤷‍♂️

Reply:
Going to remove